conans/client/build/cmake.py

```diff
@@ -306,7 +306,7 @@ def _get_cmake_definitions(self):
         except:
             pass
 
-        if self._os == "Windows" and self._compiler == "Visual Studio":
+        if str(self._os) in ["Windows", "WindowsStore"] and self._compiler == "Visual Studio":
```

Comment:
There is nothing to do in CMake to build for WindowsStore?

Reply:
right